cmd/root.go

```diff
@@ -146,10 +108,10 @@ func New() *cobra.Command {
 		}
 	}
 
-	for _, cmd := range append(cmd.Commands(), cmd) {
+	for _, subcmd := range append(cmd.Commands(), cmd) {
```

Comment:
`-Winaccurate-identifier` — we aren't just iterating over subcommands, but also over the main command; glad to find any other name to avoid a clash, but `subcmd` is slightly misleading.

Reply:
Resolved with [ff9c6a0](https://github.com/iterative/terraform-provider-iterative/pull/664/commits/ff9c6a040bd02e35f61c6e34fbce855aa8b59a40)